exasol/analytics/query_handler/query/select.py

```diff
@@ -101,17 +104,19 @@ class ModifyQuery(CustomQuery, AuditData):
     def __init__(
         self,
         query_string: str,
-        # Using DBObjectName instead of str, because counting rows of the modified table
-        # requires to use fully_qualified
-        db_object_name: DBObjectName,
+        db_object_ref: DBObjectNameWithSchema,
         db_object_type: DbObjectType | str,
         db_operation_type: DbOperationType | str,
```

Comment:
should be stored as self._db_operation_type instead of the audit_fields, because we use in the AuditLogRewritter into a SpanType

Reply:
```suggestion
        db_operation_type: DbOperationType
```